### DISCUSSION TITLE
Where is the source code for the brick game?

### DISCUSSION THREAD

**Discussion by War3Evo** [skia4delphi/skia4delphi#218] [Q&A] [answered]
I found a discussion about the game here https://learndelphi.org/check-out-this-fun-brick-game-written-in-delphi-with-skia4delphi/ but no source code.

Anyone know if this is legit and if so, where the source code is?

**Comment by viniciusfbb** [comment, marked as answer]
The source is here: https://github.com/skia4delphi/skia4delphi/tree/main/Samples/Webinar%20-%20Game/FMX

It is also included in your Skia4Delphi installation

**Comment by checkdigits** [comment]
I've updated the article to include a link to the source code now too. 😃
